app/views/layouts/application.slim

```diff
@@ -9,7 +9,10 @@ html
     = render partial: 'shared/rollbarjs'
     = javascript_pack_tag 'application'
     = javascript_include_tag 'application'
+    = javascript_include_tag '/assets/toolkit.min.js'
```

Comment:
Getting a 404 routing error when attempting to load this file.  From what I can tell this isn't checked into the repo with this commit.

Reply:
Should be fixed, moved this to be included in webpack rather than in `public/` (and not committed)